cg/apps/gt.py

```diff
@@ -1,6 +1,11 @@
 # -*- coding: utf-8 -*-
 import logging
 
+from subprocess import CalledProcessError
+import subprocess
+
+from cg.exc import CaseNotFoundError
```

Comment:
It's a trivial remark, the way imports are ordered is the following:
- build ins
- 3rd party
- internal

Can you move the cg import down to the internal imports.

Source: [PEP8](https://www.python.org/dev/peps/pep-0008/#imports)

Reply:
👍